Exercise1.java

```diff
@@ -0,0 +1,15 @@
+class Solution {
+    public List<Integer> findDisappearedNumbers(int[] nums) {
+        List<Integer> result=new ArrayList<>();
+        for(int i=1;i<nums.length+1;i++)
+        {
+            result.add(i);
+        }
+        for(int i=0;i<nums.length;i++)
+        {
+            if(result.contains(nums[i])){result.remove(new Integer(nums[i]));}
+        }
+        
+        return result;
+    }
+}
```

Comment:
Can you comment on complexity of this code (taking into account complexity of list.contains() and list.remove() )